src/Core/Layer/Layer.js

```diff
@@ -53,6 +53,13 @@ export const defineLayerProperty = function defineLayerProperty(layer, propertyN
     }
 };
 
+/**
+ * A GeometryLayer defines a geometry: globe, plane or pointcloud.
+ * This geometry can then be used to display other datas, like color layers.
+ * @constructor
+ * @param {string} id - unique id
+ * @param {Object} object3d - the THREE.Object3D will be the parent of the object hierarchy
```

Comment:
"the THREE.Object3D **that** will be the parent of the **layer's** object hierarchy"